packages/core/react/use-chat.ts

```diff
@@ -293,9 +295,11 @@ export function useChat({
   credentials,
   headers,
   body,
+  generateId,
 }: Omit<UseChatOptions, 'api'> & {
   api?: string | StreamingReactResponseAction;
 } = {}): UseChatHelpers {
+  const genId = generateId ?? nanoid;
```

Comment:
Suggestion: change `nanoid` to be the default parameter value and use `generateId` instead of `genId` (extra variable is not needed).